### PR TITLE
Added base intake class

### PR DESCRIPTION
**Pull request by ghost**
Scaffold intake class for this season

### CODE REVIEW

File: components/intake.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from magicbot import tunable
+import wpilib
+
+
+class Intake:
+    shooter_motor: wpilib.Talon
```
Comment (outsidermm) [inline components/intake.py:6]: best not to assume hardware for a skeleton

File: components/intake.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    shoot_speed = tunable(1.0)  # speed is tunable via NetworkTables
+
+    def __init__(self):
+        self.deployed = False
+
+    def deploy(self):
+        self.deployed = True
+
+    def is_ready(self):
```
Comment (outsidermm) [inline components/intake.py:15]: add magicbot feedback decorator

File: components/intake.py (hunk 1 above)
Comment (outsidermm) [inline components/intake.py:6]: plz rename adequately for your variables intake isn't shooting...

File: components/intake.py (hunk 2 above)
Comment (outsidermm) [reply components/intake.py:15]: I think, generally for intake there can be three feedback methods
1. is_note_present
2. is_fully_retracted
3. is_fully_deployed

**Review by outsidermm — COMMENTED** [review]
add syntaxical annotation for all methods

**Comment by outsidermm** [comment]
#17 I think this can help

File: components/intake.py (hunk 1 above)
Comment (auscompgeek) [reply components/intake.py:6]: agreed, this one should be simply named `motor`; you already know this is the intake

**Comment by auscompgeek** [comment]
Duplicate of #17

**Pull request closed without merging**